cake-qos-simple

```diff
@@ -76,6 +76,9 @@ start()
 	printf "\nSetting up CAKE on interface: '${ul_if}' with bandwidth: '${cake_ul_rate_Mbps}Mbit/s' and options: '${cake_ul_options}'.\n"
 	tc qdisc add dev "${ul_if}" root handle 1: cake bandwidth "${cake_ul_rate_Mbps}Mbit" ${cake_ul_options}
 
+	printf "\nSetting up tc filter to restore DSCPs from conntrack on egress packets on interface '${ul_if}'.\n"
+	tc filter add dev "${ul_if}" matchall action ctinfo dscp 63 128
```

Comment:
Should we make this a higher priority than the ECN filter below so that the ECN filter will apply second?

Reply:
I’m not sure it matters since they would be mutually exclusive of each other. Restoring DSCP doesn’t change ECN bits, or vice versa.

I haven’t used that feature since I haven’t built pedit into my custom build yet.

Reply:
It's simple enough to do though. What would be the syntax to make this 'prio 1' and the other 'prio 2'?